etest/docker/common.py

```diff
@@ -30,5 +30,5 @@
         ca_cert=os.path.join(CERT_PATH, "ca.pem"),
     )
 
-CLIENT = docker.from_env()
-API_CLIENT = docker.APIClient(base_url=BASE_URL, tls=TLS_CONFIG)
+CLIENT = docker.from_env(timeout=600)
```

Comment:
Hmm, this should probably be a parameter so the user can decide on something shorter.  Can we make the timeout five minutes and a timedelta for type safety?

Reply:
This is required so that the commit doesnt fail.

Reply:
I was afraid of that.  Let's make it a timedelta then but otherwise leave it alone.

Reply:
Its supposed to be an int. Are you sure?

Reply:
It's an int at this level because we choose that, we should use higher semantic types internal to our code and only cast on ingress and egress.  I'd say let's do it but it might have to wait until I get the types sorted out so it doesn't unintentionally break things.